osutils/utils.go

```diff
@@ -0,0 +1,39 @@
+package osutils
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rotisserie/eris"
+)
+
+var (
+	MissingEnvVarError = func(envVar string) error {
+		return fmt.Errorf("Missing %s environment variable", envVar)
+	}
+)
+
+/**
+  Returns error if the environment variable is empty
+*/
+func GetEnvE(envVar string) (string, error) {
+	env := os.Getenv(envVar)
+	if env == "" {
+		return "", MissingEnvVarError(envVar)
+	}
+	return env, nil
+}
+
+/**
+  Creates dir if it doesn't already exist
+  If the dir does exist, does nothing.
+*/
+func CreateDirIfNotExists(path string) error {
```

Comment:
from MkdirAll comment: `// If path is already a directory, MkdirAll does nothing and returns nil`...do we need this function or can we rely on MkdirAll?